components/proposal-action-card.tsx

```diff
@@ -215,6 +225,22 @@ export default function ProposalActionCard({
             </div> || ""
           }
 
+          {isPrOwner && (
+            <div className="row mt-2">
+              <ContextualSpan context="warning" classNameIcon="mb-4">
+                {t("proposal:messages.owner-pull-request")}
+              </ContextualSpan>
+            </div>
+          )}
+
+          {isProposalOwner && (
+            <div className="row mt-2">
+              <ContextualSpan context="warning">
+                {t("proposal:messages.owner-proposal")}
+              </ContextualSpan>
+            </div>
+          )}
```

Comment:
These warnings  should be visible only if the proposal passed the dispute time and it was not disputed or rejected.